explainer.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Linked Data Signatures Working Group Charter—Explainer</title>
```

Comment:
```suggestion
    <title>Linked Data Security Working Group Charter—Explainer</title>
```

Reply:
See the separate #4. Let us not change it now.